aplib.net-demo/Assets/Scripts/Inventory/AmmoPouch.cs

```diff
@@ -0,0 +1,46 @@
+using UnityEngine;
+using UnityEngine.Serialization;
+
+/// <summary>
+/// Class representing an Ammo Pouch.
+/// </summary>
+public class AmmoPouch : MonoBehaviour
+{
+    [FormerlySerializedAs("MaxAmmoCount")] [SerializeField]
+    public int maxAmmoCount;
```

Comment:
nitpicky: if you really want to comply with our standards, this public field should have a summary, though I think the name is descriptive enough